ansible/library/test_facts.py

```diff
@@ -144,7 +143,7 @@ def _read_testbed_topo_from_csv():
                         line['ptf_ipv6'], line['ptf_netmask_v6'] = \
                             _cidr_to_ip_mask(line["ptf_ipv6"])
 
-                    line['duts'] = line['dut'].translate(string.maketrans("", ""), "[] ").split(';')
+                    line['duts'] = line['dut'].translate(str.maketrans("", "", "[] ")).split(';')
```

Comment:
`str.maketrans` is not supported by python2. This change breaks our existing tests which is still using python2. That's why the PR testing failed. We may need some time to migrate to python3. By now, code changes still need to be compatible with python2.